Source/Scene/Model/ClassificationModelDrawCommand.js

```diff
@@ -128,39 +129,81 @@ const colorRenderState = {
 
 function initialize(drawCommand) {
   const command = drawCommand._command;
-  const commandList = drawCommand._commandList;
-
-  const model = drawCommand._model;
-  const useDebugWireframe = model._enableDebugWireframe && model.debugWireframe;
 
   // If debug wireframe is enabled, don't derive any new commands.
-  // Render as normal.
-  if (useDebugWireframe) {
+  // Render normally in the opaque pass.
+  if (drawCommand._useDebugWireframe) {
     command.pass = Pass.OPAQUE;
-    commandList.push(command);
+
+    drawCommand._commandListDebugWireframe = createBatchCommands(
+      drawCommand,
+      [command],
```

Comment:
Maybe use scratch variables to avoid allocating these temporary arrays every time?